### ISSUE TITLE
Minimum coverage requirement

### ISSUE THREAD

**Issue by ktalebian** [keplerproject/luacov#68]
Is it possible to enforce a minimum requirement such that running the coverage report would fail if a specific file doesn't have the minimum requirement?

**Comment by hishamhm** [comment]
What do you mean by fail? Returning an exit code other than zero?

**Comment by ktalebian** [comment]
Yes exactly so that in our CI/CD builds we can enforce minimum test coverage
